config/config.go

```diff
@@ -13,6 +13,9 @@ var mtx sync.Mutex
 // Config defines the configuration options for this service.
 type Config struct {
 	BindAddr                string `env:"BIND_ADDR"                 flag:"bind-addr"                   flagDesc:"Bind address"`
+	OfficerDatabaseUsername string `env:"OFFICER_DATABASE_USERNAME" flag:"officer-database-username"   flagDesc:"Username to access officers database"`
+	OfficerDatabasePassword string `env:"OFFICER_DATABASE_PASSWORD" flag:"officer-database-password"   flagDesc:"Password to access officers database"`
+	OfficerDatabaseUrl      string `env:"OFFICER_DATABASE_URL"      flag:"officer-database-url"        flagDesc:"URL to access officers database"`
```

Comment:
`Url` should be capitalised
```suggestion
	OfficerDatabaseURL      string `env:"OFFICER_DATABASE_URL"      flag:"officer-database-url"        flagDesc:"URL to access officers database"`
```